sunpy/instr/aia/response.py

```diff
@@ -0,0 +1,279 @@
+
+
+other important variables:
+effective area
+ion emissivity
+
+electron density
+temperature array
+
+# utilize sunpy/instr/tests/test_aia.py
+"""
+
+import numpy as np
+import pandas as pd
+from scipy.io import readsav
+
+import sunpy
+import sunpy.data.test as test
+
+
+def get_function(ion, emissivity, temperature, density, optional = None):
+    """
+    General format of functions where this is the statement of usefulness of function. If these were fits images, it would search the header for relevant information.
+
+    Parameters
+    ----------
+    variable1 : what module of import is used.
+        explanation
+    optional: (optional) string
+        A string specifying optional variable
+        e.g. strings
+
+    Returns
+    -------
+    output: tuple
+    """
+    pass
+# think: how to test
+
+def read_genx(path_to_file):
+    """
+    This definition reads the .genx output file from SolarSoft Ware (SSW). It will extract the instrument data and save them to an data file to be used for this program.
+
+     Parameters
+    ----------
+    :param path_to_file: string, the path location that leads to ssw_aia_response_genx file documents. The output file
+                    used for obtaining all of the information is:   aia_V6_all_fullinst
+    :param outfile_name: string, the name of the outfile that will store the dataframe
+                                    - could make this a dictioary called self.instr_properties in class format
+
+    Returns
+    -------
+    output: a comma separated dataframe file with arrays for each property per channel
+
+    """
+
+    # access outer dictionary from .genx file
+    ssw_dict = readsav(path_to_file)
+
+    # access inner dictionary from .genx file
+    data = ssw_dict['data']
+
+    # define wanted properties
+    indexes = ['wave', 'effarea', 'units', 'geoarea', 'scale', 'platescale', 'numfilters',  'wavemin', 'wavestep', 'wavenumsteps', 'wavelog', 'filtersincludemesh', 'meshtrans', 'usecontam', 'contamthick',  'usephottoelec','elecperev','usephottodn', 'elecperdn', 'useerror', 'fp_filter', 'ent_filter', 'primary', 'secondary', 'ccd', 'contam', 'cross_area']
+    channels = ['A94', 'A131', 'A171', 'A193', 'A211', 'A304', 'A335']
+
+    # write into dataframe for each channel from inner dictionaries
+    df = pd.DataFrame(index=indexes, columns=channels)
+
+    for channel in channels:
+
+        # pick out channel files to look into
+        for name in data.dtype.names:
+            if name.startswith('A') and name.endswith('THICK_FULL') and channel in name:
+                key = channel
+
+                # store information from those files in dataframe
+                for value in data[name]:
+                    for prop in indexes:
+                        df.loc[prop] = [value[prop][0]]
+
+    # save to dataframe outfile
+    df.to_csv('channel_properties.csv', sep=',')
+
+
+
+def load_channel_as_dict(channel = 'all'):
```

Comment:
Hi! 
Thanks for the feedback. So, I created a dataframe of the infomation needed for this project. I then thought it would be good to load the data from the file as needed by channel/wavelength. The load_channel could then be used throughout the program to quickly pick out information as needed.  I am wondering if a class structure that just makes these self.a94 = dict['info'] for example would be better for accessing the properties globally. Thoughts?